cmd/monaco/integrationtest/v2/settings_integration_test.go

```diff
@@ -437,6 +424,44 @@ func TestOrdered_InsertAtFrontAndBackWorks(t *testing.T) {
 	})
 }
 
+func filterObjectsForScope(pgiMeId string) func(object dtclient.DownloadSettingsObject) bool {
+	return func(object dtclient.DownloadSettingsObject) bool {
+		return object.Scope == pgiMeId
+	}
+}
+
+func createSettingsClient(t *testing.T, env manifest.EnvironmentDefinition, opts ...func(dynatraceClient *dtclient.SettingsClient)) client.SettingsClient {
+
+	clientFactory := clients.Factory().
+		WithOAuthCredentials(clientcredentials.Config{
+			ClientID:     env.Auth.OAuth.ClientID.Value.Value(),
+			ClientSecret: env.Auth.OAuth.ClientSecret.Value.Value(),
+			TokenURL:     env.Auth.OAuth.GetTokenEndpointValue(),
+		}).
+		//WithConcurrentRequestLimit(concurrentRequestLimit).
```

Comment:
nit: While I see it, can we remove this or uncomment it?

Reply:
Removed